x/assetprofile/types/codec.go

```diff
@@ -31,18 +31,18 @@ func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 }
 
 var (
-	Amino     = codec.NewLegacyAmino()
-	ModuleCdc = codec.NewAminoCodec(Amino)
+	amino     = codec.NewLegacyAmino()
```

Comment:
is there a reason we use `animo` and not `Amino` but `ModuleCdc` ?

Reply:
Just following Cosmos-sdk standard practice. Amino variable should be non-exportable and to use it we should use ModuleCdc
Reference: https://github.com/cosmos/cosmos-sdk/blob/v0.47.11/x/feegrant/codec.go